.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!-- IMPORTANT!
+  - Every PR must reference an issue; this helps to explain the intent of the PR
+ -->
+
+Fixes #
+
+# 👩🏻‍💻 What does this PR do? 
+ <!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
+
+# 🧪 How has this change been tested? 
```

Comment:
```suggestion
# 🧪 How has this change been tested? / test instructions
```

Reply:
Have changed the wording around this: https://github.com/openmsupply/open-msupply/pull/1174/commits/d0356477a9e2f38ec705a8e7cefbbcc5306005fd